lib/postProcessLdesConceptualService.js

```diff
@@ -392,3 +393,51 @@ async function flagInstancesModifiedConcept(vGraph, vService, service) {
 
   await updateSudo(updateQueryStr);
 }
+
+async function ensureConceptDisplayConfigs(versionedServiceGraph, versionedService, conceptualService) {
+  // This list limits the type of bestuurseenheden for which the config objects will be created.
+  // Only types that have access to the LPDC module should be added here.
+  const ALLOWED_BESTUURSEENHEID_CLASSIFICATIONS = [
+    'http://data.vlaanderen.be/id/concept/BestuurseenheidClassificatieCode/5ab0e9b8a3b2ca7c5e000001' // Gemeente
+  ];
+
+  const insertConfigsQuery = `
+    ${PREFIXES}
+    
+    INSERT {
+      GRAPH ?graph {
+        ?concept lpdcExt:hasConceptDisplayConfiguration ?configUri .
+        ?configUri a lpdcExt:ConceptDisplayConfiguration ;
+          mu:uuid ?configId ;
+          lpdcExt:conceptIsNew "true"^^xsd:boolean ;
+          lpdcExt:conceptInstantiated "false"^^xsd:boolean ;
+          dct:relation ?eenheid .
+      }
+    }
+    WHERE {
+      VALUES ?eenheidClassificatie {
+        ${ALLOWED_BESTUURSEENHEID_CLASSIFICATIONS.map(sparqlEscapeUri).join(' ')}
+      }
+    
+      ?eenheid a besluit:Bestuurseenheid ;
+        mu:uuid ?eenheidId ;
+        besluit:classificatie ?eenheidClassificatie .
+    
+      BIND(IRI(CONCAT("http://mu.semte.ch/graphs/organizations/", STR(?eenheidId), "/LoketLB-LPDCGebruiker")) as ?graph)
+      BIND(${sparqlEscapeUri(conceptualService)} as ?concept)
+    
+      GRAPH ?graph {
+        FILTER NOT EXISTS {
+          ?concept lpdcExt:hasConceptDisplayConfiguration ?configUri .
+          ?configUri dct:relation ?eenheid .
+        }
+      }
+    
+      ${/*this is a bit of trickery to generate UUID and URI's since STRUUID doesn't work properly in Virtuoso: https://github.com/openlink/virtuoso-opensource/issues/515#issuecomment-456848368 */''}
```

Comment:
that's an elegant way to include comments for mu-auth to tolerate :-)
Side note; i would have preferred generating with JS the uri's. But anyway this should work too.

Reply:
> Side note; i would have preferred generating with JS the uri's. But anyway this should work too.

How would that work exactly?